### PR TITLE
refactor: Secure MessageBus Rework

### PR DESCRIPTION
**Pull request by lenny-intel**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/device-virtual-go/blob/master/.github/CONTRIBUTING.md

## What is the new behavior?
Latest SDK which removed shared code which forced ZMQ dependency on all services.
Added Missing InscureSecrets

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## New Imports

- [ ] Yes
- [x] No

### CODE REVIEW

File: cmd/res/configuration.toml [hunk 1]
```diff
@@ -14,7 +21,7 @@ ConnectRetries = 10
 Labels = []
 EnableAsyncReadings = true
 AsyncBufferSize = 16
-UseMessageBus = false
+UseMessageBus = true
```
Comment (hahattan) [inline cmd/res/configuration.toml:24]: Is this for demo purpose since other device services are defaulting to `false` ?

**Review by hahattan — COMMENTED** [review]
have one concern but LGTM overall

**Comment by cloudxxx8** [comment]
@lenny-intel I know we will make the default `UseMessageBus = true`
May we modify all of them after the core-data can receive from Message Bus?

**Review by cloudxxx8 — APPROVED** [review]
LGTM

**Comment by lenny-intel** [comment]
> I know we will make the default `UseMessageBus = true`
> May we modify all of them after the core-data can receive from Message Bus?

@cloudxxx8 , @hahattan , correct. I didn't mean to leave it enabled. Fixing now.

File: cmd/res/configuration.toml (hunk 1 above)
Comment (lenny-intel) [reply cmd/res/configuration.toml]: Fixed

**Review by cloudxxx8 — APPROVED** [review]
LGTM

**Pull request merged**